scripts/tablegen.py

```diff
@@ -120,17 +131,29 @@ def decode_helm_yaml(s):
     for d in data:
         print d
         if "<!-- AUTO-GENERATED-START -->" in d:
-            print '| Key | Default Value | Description |'
-            print '| --- | --- | --- |'
             break
 
     with open('values.yaml', 'r') as f_v:
         d_v = f_v.read()
+
+        # transform values.yaml into a encoded string dictionary
         yaml = YAML()
         code = yaml.load(d_v)
         yaml.explicit_start = True
         yaml.dump(code, sys.stdout, transform=decode_helm_yaml)
 
+        # Order the encoded string dictionary
+        od = collections.OrderedDict(sorted(prdict.items(), key=lambda t: t[0]))
+
+        # Print encoded string dictionary
+        for k, v in od.items():
+            print ("## `%s` configuration options\n" % k)
```

Comment:
I'd drop the 'configuration' string in there, it's redundant and causes too much wrapping in the right-hand TOC.

Reply:
done.